streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1565,6 +1565,14 @@ public String topic() {
                 return null;
         }
 
+        @Override
+        public Class<? extends TopicNameExtractor> topicNameExtractor() {
+            if (topicNameExtractor instanceof StaticTopicNameExtractor)
+                return null;
+            else
+                return topicNameExtractor.getClass();
```

Comment:
Can you please add `{ }` to both blocks? (Please also add them in `topic()` method above).